gemini/use-cases/retrieval-augmented-generation/code_rag.ipynb

```diff
@@ -0,0 +1,1231 @@
+{
```

Comment:
<p>This section seems very useful, but I would recommend moving into a separate notebook or a full sample app, and just showing a basic use case with the Code RAG in this notebook.</p><p> </p><p>Assitant → Assistant</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/481/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
<p>Fixed <code>Assitant → Assistant</code>.</p><p> </p><p>This class makes it easy to use LLM with a prompt config. </p><p>Using this we can manage all our prompts and their associated model parameters together.</p>

Reply:
<p>Ok</p>